core/src/main/java/com/google/errorprone/bugpatterns/javadoc/InvalidLink.java

```diff
@@ -156,7 +156,7 @@ public Void visitLink(LinkTree linkTree, Void unused) {
         element =
             JavacTrees.instance(state.context)
                 .getElement(new DocTreePath(getCurrentPath(), linkTree.getReference()));
-      } catch (NullPointerException e) {
+      } catch (NullPointerException | AssertionError e) {
         // TODO(b/176098078): remove once JDK 12 is the minimum supported version
         // https://bugs.openjdk.java.net/browse/JDK-8200432
```

Comment:
If this pertains to another javac bug (perhaps solved post JDK 12?), should it be referenced here?